.github/workflows/fuzz.yml

```diff
@@ -1,6 +1,15 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  # Trigger the workflow on push or pull requests, but only for the
+  # master branch
+  push:
+    branches:
+      - master
+      - '*/ci'
+  pull_request:
+    branches:
+      - master
```

Comment:
I think you can include '*/ci' here aswell to make sure that remote '*/ci' branches are tested if used for a PR. At least on Azure Pipelines the branch list is about the remote repository (PR source, not target).